### DISCUSSION TITLE
Frontend API: open note in a new split

### DISCUSSION THREAD

**Discussion by WPum** [TriliumNext/Trilium#2891] [Q&A] [answered]
Is there any frontend API providing the feature "open in a new split"?

**Comment by zadam** [comment, marked as answer]
Hi, added `api.openSplitWithNote(notePath, activate)` method into the API, will be available from 0.52.1 

https://zadam.github.io/trilium/frontend_api/FrontendScriptApi.html

**Reply by WPum** [reply]
Thanks!
